File: suites/05_ml_primitives/comparisons/convolution.go
package main
import "fmt"
func convScore(w, h int64) int64 {
	var total int64
	for y := int64(1); y < h-1; y++ {
		for x := int64(1); x < w-1; x++ { total += ((x * 3) + (y * 5)) % 97 }
	}
	return total
}
func main() { fmt.Println(convScore(512, 512)) }

File: suites/05_ml_primitives/comparisons/convolution.rs
fn conv_score(w: i64, h: i64) -> i64 {
    let mut total: i64 = 0;
    for y in 1..h-1 { for x in 1..w-1 { total += ((x * 3) + (y * 5)) % 97; } }
    total
}
fn main() { println!("{}", conv_score(512, 512)); }
